airflow/versions.py

```diff
@@ -2,41 +2,33 @@
     {
         "product": "2.6.3",
         "python": "3.9",
-        "git_sync": "v4.2.3",
+        "git_sync": "v4.2.4",
         "statsd_exporter": "0.26.1",
         "tini": "0.19.0",
         "vector": "0.39.0",
     },
     {
-        "product": "2.7.2",
-        "python": "3.9",
-        "git_sync": "v4.2.3",
-        "statsd_exporter": "0.26.1",
-        "tini": "0.19.0",
-        "vector": "0.39.0",
-    },
-    {
-        "product": "2.7.3",
+        "product": "2.8.1",
         "python": "3.9",
-        "git_sync": "v4.2.3",
+        "git_sync": "v4.2.4",
         "statsd_exporter": "0.26.1",
         "tini": "0.19.0",
         "vector": "0.39.0",
     },
     {
-        "product": "2.8.1",
+        "product": "2.8.4",
         "python": "3.9",
-        "git_sync": "v4.2.3",
+        "git_sync": "v4.2.4",
         "statsd_exporter": "0.26.1",
         "tini": "0.19.0",
         "vector": "0.39.0",
     },
     {
-        "product": "2.8.3",
+        "product": "2.9.2",
         "python": "3.9",
-        "git_sync": "v4.2.3",
+        "git_sync": "v4.2.4",
         "statsd_exporter": "0.26.1",
         "tini": "0.19.0",
         "vector": "0.39.0",
-    },
+    },    
```

Comment:
trailing spaces (will be picked up in #765)

```suggestion
    },
```